persistence-cassandra/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/cassandra/CassandraReadSideHandler.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.scaladsl.persistence.cassandra
+
+import akka.stream.ActorAttributes
+import akka.stream.scaladsl.Flow
+import akka.{ Done, NotUsed }
+import com.datastax.driver.core.{ BatchStatement, BoundStatement }
+import com.lightbend.lagom.scaladsl.persistence.ReadSideProcessor.ReadSideHandler
+import com.lightbend.lagom.scaladsl.persistence._
+import com.lightbend.lagom.scaladsl.persistence.cassandra.CassandraSession
+import org.slf4j.LoggerFactory
+
+import scala.collection.immutable.Seq
+import scala.concurrent.{ ExecutionContext, Future }
+
+/**
+ * Internal API
+ */
+private[cassandra] abstract class CassandraReadSideHandler[Event <: AggregateEvent[Event], Handler](
+  session:    CassandraSession,
+  handlers:   Map[Class[_ <: Event], Handler],
+  dispatcher: String
+)(implicit ec: ExecutionContext) extends ReadSideHandler[Event] {
+
+  private val log = LoggerFactory.getLogger(this.getClass)
```

Comment:
Seems like the slf4j LoggerFactory is used throughout and explicitly in Logging.html in the docs so I don't think so.